<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QWidget" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>841</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="classmatestab">
    <attribute name="title">
     <string>Classmates</string>
    </attribute>
    <widget class="QTableView" name="classmatesTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="friendstab">
    <attribute name="title">
     <string>Friends</string>
    </attribute>
    <widget class="QTableView" name="friendsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="colleaguestab">
    <attribute name="title">
     <string>Colleagues</string>
    </attribute>
    <widget class="QTableView" name="colleaguesTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="relativestab">
    <attribute name="title">
     <string>Relatives</string>
    </attribute>
    <widget class="QTableView" name="relativesTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="teacherstab">
    <attribute name="title">
     <string>Teachers</string>
    </attribute>
    <widget class="QTableView" name="teachersTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="superiorstab">
    <attribute name="title">
     <string>Superiors</string>
    </attribute>
    <widget class="QTableView" name="superiorsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="clientstab">
    <attribute name="title">
     <string>Clients</string>
    </attribute>
    <widget class="QTableView" name="clientsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="otherstab">
    <attribute name="title">
     <string>Others</string>
    </attribute>
    <widget class="QTableView" name="othersTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="commitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdrawButtion">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QLabel" name="hintlabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cannot withdraw after committing</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
